src/main/java/com/adyen/model/checkout/CheckoutPaymentsAction.java

```diff
@@ -595,12 +638,17 @@ public boolean equals(Object o) {
                 Objects.equals(this.totalAmount, checkoutPaymentsAction.totalAmount) &&
                 Objects.equals(this.type, checkoutPaymentsAction.type) &&
                 Objects.equals(this.entity, checkoutPaymentsAction.entity) &&
+                Objects.equals(this.sdkData, checkoutPaymentsAction.sdkData) &&
+                Objects.equals(this.resendInterval, checkoutPaymentsAction.resendInterval) &&
+                Objects.equals(this.resendMaxAttempts, checkoutPaymentsAction.resendMaxAttempts) &&
+                Objects.equals(this.resendUrl, checkoutPaymentsAction.resendUrl) &&
+                Objects.equals(this.redirect, checkoutPaymentsAction.redirect) &&
                 Objects.equals(this.url, checkoutPaymentsAction.url);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(alternativeReference, data, downloadUrl, expiresAt, initialAmount, instructionsUrl, issuer, maskedTelephoneNumber, merchantName, merchantReference, method, paymentData, paymentMethodType, qrCodeData, reference, shopperEmail, shopperName, surcharge, token, totalAmount, type, url, entity);
+        return Objects.hash(alternativeReference, data, downloadUrl, expiresAt, initialAmount, instructionsUrl, issuer, maskedTelephoneNumber, merchantName, merchantReference, method, paymentData, paymentMethodType, qrCodeData, reference, shopperEmail, shopperName, surcharge, token, totalAmount, type, url, entity, sdkData);
```

Comment:
There are 5 new fields and only 1 field is added in hashcode()

Reply:
Added.